client/components/Login.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { checkUser as checkUserActionCreator } from '../store';
+import { checkUser as checkUserActionCreator } from '../store/index';
```

Comment:
The original line 3 that imports from '../store' would've worked too-- the index.js file is implicitly imported. 

This applies to imports of '../store' in other files as well.

Reply:
Good point.  Changed them all.